pydantic_ai_slim/pydantic_ai/_output.py

```diff
@@ -1068,7 +1079,7 @@ async def call_tool(
     ) -> Any:
         output = await self.processors[name].call(tool_args, ctx, wrap_validation_errors=False)
         for validator in self.output_validators:
-            output = await validator.validate(output, ctx, wrap_validation_errors=False)
+            output = await validator.validate(output, ctx, wrap_validation_errors=False, partial=ctx.tool_args_partial)
```

Comment:
Now that we're putting this on the `RunContext` and we require the output validator function to take context, we don't really need the new argument anymore. What do you think about just documenting that the function can access `ctx.tool_args_partial`?

Edit: Although I suppose `tool_args` is not really accurate if we're using an [output mode](https://ai.pydantic.dev/output/#output-modes) other than tool output... What do you think about calling it `partial_output`?